yarn-lock.js

```diff
@@ -0,0 +1,62 @@
+const fs = require('fs')
+const lockfile = require('@yarnpkg/lockfile')
+const chalk = require('chalk')
+
+const yarnLock = 'yarn.lock'
+const yarnRegistryRe = /https:\/\/registry.yarnpkg.com/
+
+let ernDirs = fs.readdirSync(process.cwd()).filter(x => x.startsWith('ern'))
+ernDirs.push(yarnLock) // Root yarn.lock
+
+ernDirs.forEach((current) => {
```

Comment:
minor: parenthesis are not needed for an arrow function with one param (`(current)` can just be `current`)